types/ember/test/ember-module-tests.ts

```diff
@@ -0,0 +1,280 @@
+import Ember from 'ember';
+import EngineInstance from '@ember/engine/instance';
+import Component from '@ember/component';
+import EmberObject from '@ember/object';
+import ComputedProperty from '@ember/object/computed';
+import Evented from '@ember/object/evented';
+// $
+Ember.$; // $ExpectType JQueryStatic
+// A
+Ember.A(); // $ExpectType NativeArray<{}>
+Ember.A([1, 2]); // $ExpectType NativeArray<number>
+// addListener
+Ember.addListener({ a: 'foo' }, 'a', {}, () => {});
+Ember.addListener({ a: 'foo' }, 'a', null, () => {});
+// addObserver
+Ember.addObserver({ a: 'foo' }, 'a', null, () => {});
+Ember.addObserver({ a: 'foo' }, 'a', {}, () => {});
+// aliasMethod
+Ember.aliasMethod('init');
+// assert
+Ember.assert('2+2 should always be 4', 2 + 2 === 4);
+// assign
+const o1 = Ember.assign({ a: 1 }, { b: 2 });
+o1.a; // $ExpectType number
+o1.b; // $ExpectType number
+o1.c; // $ExpectError
+// Ember.bind // $ExpectError
+// cacheFor
+Ember.cacheFor({ a: 123 }, 'a'); // $ExpectType number | undefined
+Ember.cacheFor({ a: 123 }, 'x'); // $ExpectError
+// compare
+Ember.compare('31', '114'); // $ExpectType number
+// copy
+Ember.copy({ a: 12 }, true).a; // $ExpectType number
+Ember.copy({ a: 12 }); // $ExpectType any
+Ember.copy({ a: 12 }).a; // $ExpectType any
+Ember.copy({ a: 12 }).b; // $ExpectType any
+// debug
+Ember.debug('some info for developers');
+// deprecate
+Ember.deprecate("you shouldn't use this anymore", 3 === 3, {
+    id: 'no-longer-allowed',
+    until: '99.0.0'
+});
+// get
+Ember.get({ z: 23 }, 'z'); // $ExpectType number
+Ember.get({ z: 23 }, 'zz'); // $ExpectError
+// getEngineParent
+Ember.getEngineParent(new EngineInstance()); // $ExpectType EngineInstance
+// getOwner
+Ember.getOwner(new Component());
```

Comment:
needs an $ExpectType

Reply:
unfortunately `getOwner` is currently an `any`. Asserting this type is not all that useful